tests/acceptance/test_project_information.py

```diff
@@ -0,0 +1,298 @@
+
+        project_name = "Test Project - Leipzig Test Intersection - OTCamera19"
+        date_value = "2023-05-24"
+        time_value = "06:00:00"
+
+        # Locate inputs via aria-labels
+        name_sel = (
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_PROJECT_NAME)}"]'
+        )
+        date_sel = (
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_DATE)}"]'
+        )
+        time_sel = (
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_TIME)}"]'
+        )
+
+        _set_input_value(page, name_sel, project_name)
+        _set_input_value(page, date_sel, date_value)
+        _set_input_value(page, time_sel, time_value)
+
+        # Assertions
+        assert page.locator(name_sel).input_value() == project_name
+        assert page.locator(date_sel).input_value() == date_value
+        assert page.locator(time_sel).input_value() == time_value
+
+    @pytest.mark.timeout(300)
+    @pytest.mark.playwright
+    @pytest.mark.xfail(
+        reason=(
+            "German date format not supported by date input; component expects ISO format"  # noqa
+        )
+    )
+    @pytest.mark.usefixtures("external_app")
+    def test_project_information_german_format_playwright(
+        self,
+        page: Page,
+        external_app: NiceGUITestServer,
+        resource_manager: ResourceManager,
+    ) -> None:
+        """Attempt to set German date format (expected to fail, mirrors Selenium test)."""  # noqa
+        base_url = getattr(external_app, "base_url", "http://127.0.0.1:8080")
+        page.goto(base_url + ENDPOINT_MAIN_PAGE)
+
+        project_name = "Test Project - Leipzig Test Intersection - OTCamera19"
+        date_value = "24.05.2023"  # German format
+        time_value = "06:00:00"
+
+        name_sel = (
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_PROJECT_NAME)}"]'
+        )
+        date_sel = (
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_DATE)}"]'
+        )
+        time_sel = (
+            f'[aria-label="{resource_manager.get(ProjectKeys.LABEL_START_TIME)}"]'
+        )
+
+        _set_input_value(page, name_sel, project_name)
+        _set_input_value(page, date_sel, date_value)
+        _set_input_value(page, time_sel, time_value)
+
+        assert page.locator(name_sel).input_value() == project_name
+        # Expected to fail on the following assertion per xfail reason
+        assert page.locator(date_sel).input_value() == date_value
+        assert page.locator(time_sel).input_value() == time_value
+
+    @pytest.mark.timeout(300)
+    @pytest.mark.playwright
+    @pytest.mark.usefixtures("external_app")
+    def test_project_information_export_import_playwright(
+        self,
+        page: Page,
+        external_app: NiceGUITestServer,
+        resource_manager: ResourceManager,
+        test_data_tmp_dir: _Path,
+        monkeypatch: Any,
```

Comment:
I assume monkeypatch an all code related to it will not work, because the server ist started as a separate process. Why do we need monkeypatch here? There is a helper available to select a file in the file picker.